<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="starList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLine"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Right Ascension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RALine"/>
          </item>
          <item>
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>Declination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="declinationLine"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Diameter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="diameterLine"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Image location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imageLine"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Add Star</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Show from my constellations only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewButton">
            <property name="text">
             <string>View Star</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="searchLine">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Page">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConstellations">
    <property name="title">
     <string>Constellations</string>
    </property>
    <addaction name="viewConstellation"/>
   </widget>
   <addaction name="menuMain_Page"/>
   <addaction name="menuConstellations"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="viewConstellation">
   <property name="text">
    <string>View constellations</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
